confirm/command.go

```diff
@@ -34,10 +35,21 @@ func (o Options) Run() error {
 
 	if m.(model).aborted {
 		os.Exit(exit.StatusAborted)
-	} else if m.(model).confirmation {
-		os.Exit(0)
+
 	} else {
-		os.Exit(1)
+		if o.ShowOutput {
+			confirmationText := m.(model).negative
+			if m.(model).confirmation {
+				confirmationText = m.(model).affirmative
+			}
+			fmt.Println(m.(model).prompt, confirmationText)
+		}
+
+		if m.(model).confirmation {
+			os.Exit(0)
+		} else {
+			os.Exit(1)
+		}
 	}
```

Comment:
Here you should remove the else and outdent the code and also the last else statement can be outdented as well.

The code will look something like this.
```
	}

	if o.ShowOutput {
		confirmationText := m.(model).negative
		if m.(model).confirmation {
			confirmationText = m.(model).affirmative
		}
		fmt.Println(m.(model).prompt, confirmationText)
	}

	if m.(model).confirmation {
		os.Exit(0)
	}

	os.Exit(1)
```

Reply:
I must have missed the CI result notifications, really sorry.
This should be fixed as per you suggestions, thanks!

Regarding the model structure, do you think [this commit](https://github.com/charmbracelet/gum/pull/427/commits/432182f4b8eefa0aa006daa8c672a15d8da4beb9) is necessary?